src/modalities/__main__.py

```diff
@@ -328,49 +332,56 @@ def run(self, components: TrainingComponentsInstantiationModel):
         components.evaluation_subscriber.consume_dict({"No. Parameters": num_params})
         logging.info(f"Training model with {num_params} parameters.")
 
-        if len(components.settings.training.activation_checkpointing_modules) > 0:
```

Comment:
why did we remove activation checkpointing here?

Reply:
it is now a component, where we make sure that it is the required FSDP-wrapped model. The component is also unit tested now.